chingu-voyage4/Geckos-Team-32
# Database

## Purpose
* started coding out the database with @marylicious 

## Approach/Fixes
* set up structure for database and server-side routes
* imported necessary dependencies

## Testing
* download this PR to your local machine
* check to see that you can create a user account
* try making the same user login after successfully making it before

##Notes
* This is really basic so far
